readme.md

```diff
@@ -1,6 +1,6 @@
 # noop-process [![Build Status](https://travis-ci.org/sindresorhus/noop-process.svg?branch=master)](https://travis-ci.org/sindresorhus/noop-process)
 
-> Create a noop process and get the PID
+> Create a noop (no operation) process and get the PID
```

Comment:
You need to update package.json too.

Reply:
@sindresorhus  Done. Did a quick fix to the author's url too.